app/src/main/java/com/losextraditables/bu/instagrammers/view/holder/UserViewHolder.java

```diff
@@ -2,37 +2,60 @@
 
 
 import android.content.Context;
+import android.graphics.Color;
+import android.graphics.PorterDuff;
 import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.daimajia.androidanimations.library.Techniques;
+import com.daimajia.androidviewhover.BlurLayout;
 import com.losextraditables.bu.R;
 import com.losextraditables.bu.instagrammers.view.model.InstagrammerModel;
 import com.squareup.picasso.Picasso;
 
 import butterknife.Bind;
 import butterknife.ButterKnife;
+import de.hdodenhof.circleimageview.CircleImageView;
 
 public class UserViewHolder extends RecyclerView.ViewHolder {
 
     private final Context context;
-    @Bind(R.id.instagrammer_picture)
+    @Bind(R.id.user_blur_image)
     ImageView picture;
-    @Bind(R.id.instagrammer_name)
-    TextView name;
-    @Bind(R.id.instagrammer_username)
+    @Bind(R.id.blur_layout)
+    BlurLayout blurLayout;
+
+    View hover;
     TextView username;
+    ImageView avatar;
 
     public UserViewHolder(View itemView, Context context) {
         super(itemView);
         ButterKnife.bind(this, itemView);
         this.context = context;
+        buildHoverView();
+        picture.setColorFilter(Color.argb(150, 155, 155, 155), PorterDuff.Mode.SRC_ATOP);
+    }
+
+    private void buildHoverView() {
+        hover = LayoutInflater.from(context).inflate(R.layout.instagrammer_hover_item_list, null);
+        username = (TextView) hover.findViewById(R.id.user_name);
+        avatar =(CircleImageView) hover.findViewById(R.id.user_avatar);
+        blurLayout.setHoverView(hover);
+        blurLayout.addChildAppearAnimator(hover, R.id.user_name, Techniques.FadeInUp);
```

Comment:
The same I told you before, can you please don't use the word "user" as long as possible please?